src/main/cpp/utils/logging/debug/Logger.cpp

```diff
@@ -357,7 +357,7 @@ void Logger::SetLoggingLevel(
     m_level = level;
 }
 
-Logger::Logger() : m_option(LOGGER_OPTION::EAT_IT),
+Logger::Logger() : m_option(LOGGER_OPTION::DASHBOARD),
```

Comment:
Change this back to Eat It

Reply:
Yeah, this file shouldn't have any changes